src/modules/files/services/file-upload.service.ts

```
@@ -0,0 +1,107 @@
+      if (!mime || !acceptedMimeTypes.includes(mime)) {
+        log.warn({ mime }, 'Unsupported file type detected');
+        throw createHttpError({
+          status: 400,
+          message: 'Unsupported file type',
+        });
+      }
+      log.info({ mime, size: file.size }, 'File type and size are valid');
+
+      const safeName = String(file.originalname || '')
+        // eslint-disable-next-line no-control-regex, no-useless-escape
+        .replace(/[\/\\\u0000-\u001F]/g, '')
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)